oneandone/resource_oneandone_server.go

```diff
@@ -282,7 +299,15 @@ func resourceOneandOneServerRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("name", server.Name)
 	d.Set("datacenter", server.Datacenter.CountryCode)
 
-	d.Set("hdds", readHdds(server.Hardware))
+	if server.Hardware.FixedInsSizeId != "" {
+		fixedsize, err := Id2fixedsize(config, server.Hardware.FixedInsSizeId)
+		if err != nil {
+			return err
+		}
+		d.Set("fixedsize", fixedsize)
+	} else {
+		d.Set("hdds", readHdds(server.Hardware))
+	}
```

Comment:
It seems that you can't have multiple disks when `FixedInsSizeId` is passed in the request.

Funny it didn't error out.
```
[
		{
			"id": "F364701B6EEB47ED4C90B02B979F42C1",
			"name": "Random S Server",
			"datacenter": {
				"id": "4EFAD5836CE43ACA502FD5B99BEE44EF",
				"country_code": "DE",
				"location": "Germany"
			},
			"status": {
				"state": "POWERED_OFF",
				"percent": 0
			},
			"hardware": {
				"vcore": 1,
				"cores_per_processor": 1,
				"ram": 0.5,
				"hdds": [
					{
						"id": "8B8A878F868891EF6F93D6BFDC71409F",
						"size": 30,
						"is_main": true
					}
				],
				"fixed_instance_size_id": "5D95EED01D24E06706B5A60F428725DE"
			},
			"image": {
				"id": "B5F778B85C041347BCDCFC3172AB3F3C",
				"name": "centos7-64std"
			},
			"ips": [
				{
					"id": "BD033522A46075E091D3CE067AD5B79E",
					"ip": "212.227.8.108"
				}
			]
		}
]
```